### PR TITLE
added .gitignore file, removed .DS_Store file

### PR DESCRIPTION
**Pull request by austramarta**
- Have removed .DS_Store file
- Have added .gitignore file
- Included .DS_Store filename within the .gitignore

Do let me know if I have f.. messed something up!

### CODE REVIEW

File: visual.js [hunk 1]
```diff
@@ -81,7 +81,6 @@ function drawPiece(piece) {
         "B": "Bishop",
         "P": "Pawn"
     }
-    //logic to make which piece is necessary
```
Comment (klepzers) [inline visual.js]: Why this is removed in this branch?

**Review by klepzers — CHANGES_REQUESTED** [review]

File: visual.js (hunk 1 above)
Comment (austramarta) [reply visual.js]: my mistake. have added it back

Comment (klepzers) [reply visual.js]: @austramarta please push the changes :)
Also - you can check changes like this (after committing) by `git diff HEAD~~..HEAD`

**Review by Toms-R — APPROVED** [review]

**Review by klepzers — APPROVED** [review]
Verī naais.

**Pull request merged**